marshmallow/utils.py

```diff
@@ -343,3 +343,15 @@ def callable_or_raise(obj):
 def get_func_args(func):
     """Return a tuple of argument names for a function."""
     return inspect.getargspec(func).args
+
+
+def get_callable_args(func):
```

Comment:
I dont guess so. I was really just following how there was ```get_func_name``` and ```get_callable_name``` with the former being deprecated, but I dont see why all cases of ```get_func_args``` couldnt be replaced with ```get_callable_args```. Ill give that a try.